packages/graph-explorer/src/routes/Settings/SettingsGeneral.tsx

```diff
@@ -113,9 +115,11 @@ export default function SettingsGeneral() {
           description="Logs the generated database queries to the servers logger. If you have encountered an issue this might be helpful with diagnosing the root cause."
         />
         <ImportantBlock>
-          This <b>will not</b> log any data returned by the database queries.
-          However, the node & edge labels, ID values, and any value filters will
-          be present in the queries.
+          <p>
+            This <b>will not</b> log any data returned by the database queries.
+            However, the node & edge labels, ID values, and any value filters
+            will be present in the queries.
```

Comment:
does this mean they will not be included in queries if you turn the logging option on?

Reply:
No. It means that if the logging is turned on, then those values will be logged as part of the query.

The point of the message is to inform the user that the query might contain some "customer data" such as IDs and filters.